docs/getting-started-guides/docker.md

```diff
@@ -0,0 +1,71 @@
+## Running kubernetes locally via Docker
```

Comment:
Did you test this on your mac as well as on your linux box? It'd be nice to know it worked both with a local docker install and with boot2docker.